airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2566 @@
+version: "0.44.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["{{ parameters.extractor_path }}"]
```

Comment:
What is the purpose of this parameter? I see all stream have `extractor_path: "data"`